lib/Test2/Compare/Custom.pm

```diff
@@ -67,6 +74,7 @@ provides a way for you to write custom checks for fields in deep comparisons.
             my %args = @_;
             return $args{got} ? 1 : 0;
         },
+        stringify_got => 1,
```

Comment:
I think this would be cleaner if you put this above the `code => ` argument instead of below it.